pegasus/routes/dev_routes.rb

```diff
@@ -27,3 +30,12 @@
     }
   )
 end
+
+post '/api/dev/set-last-dtt-green' do
+  forbidden! unless rack_env == :test
+  dont_cache
+  forbidden! unless params[:token] == CDO.slack_set_last_dtt_green_token
+
+  sha = GitHub.sha('test')
+  InfraTestTopic.set_green_commit(sha)
```

Comment:
That is handled by the DOTD script, though so is updating the Slack#infra-test topic. My vision for this API route usage is solely as a slack slash command. From that vantage point,  don't think updating `DTT: yes` is appropriate.